README.md

```diff
@@ -1,7 +1,7 @@
-About ndarray_listener
+About ndarray-listener
```

Comment:
Not sure if you edit this manually or not but I though I should let you know that  you can do that with `conda smithy rerender`.